sdk-api-src/content/cfapi/ns-cfapi-cf_placeholder_create_info.md

```diff
@@ -56,7 +56,9 @@ Contains placeholder information for creating new placeholder files or directori
 
 ### -field RelativeFileName
 
-The name of the child placeholder file or directory to be created.
+The name of the child placeholder file or directory to be created. 
```

Comment:
The name of the placeholder file or directory to be created.
This parameter must not contain a relative path or else CfCreatePlaceholders will fail with ERROR_CLOUD_FILE_INVALID_REQUEST.
For example if the sync root of the provider is c:\SyncRoot and a placeholder named "pl.txt" is being created in a child directory of the sync root such as c:\SyncRoot\ChildDir, then BaseDirectoryPath = c:\SyncRoot\ChildDir and RelativeFileName = pl.txt is expected. It is an error to specify BaseDirectoryPath = c:\SyncRoot and RelativeFileName = child\pl.txt.